mtags/src/main/scala-2/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -167,6 +167,11 @@ class MetalsGlobal(
       visit: Member => Boolean
   ): SymbolSearch.Result = {
 
+    def isRelevantWorkspaceSymbol(sym: Symbol): Boolean = {
+      pprint.log(sym)
+      sym.isClass || sym.isModule || sym.isPackageObject || sym.isTraitOrInterface
```

Comment:
```suggestion
      sym.isClass || sym.isModule || sym.isPackageObject || sym.isTrait || sym.isInterface
```
looks to compile in the Scala 2.11.12. Though I wonder if we could add a parameter in search to return particular kinds?